.github/workflows/integration-tests.yml

```diff
@@ -123,3 +123,105 @@ jobs:
           flags: ${{ steps.test-coverage-flags.outputs.os }},${{ steps.test-coverage-flags.outputs.node }}
           token: ${{ secrets.CODECOV_TOKEN }}
         if: '${{ !steps.release-check.outputs.IS_RELEASE }}'
+  integration-win-node-23:
+    name: Integration test windows latest node23 specific
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 40
+    strategy:
+      matrix:
+        os: [windows-latest]
+        node-version: ['23']
```

Comment:
Is this not `latest`?

Reply:
I want to be explicit in it being node 23 :)